Marlin/src/pins/stm32f1/pins_FYSETC_AIO_II.h

```diff
@@ -24,13 +24,13 @@
 #include "env_validate.h"
 
 #define BOARD_INFO_NAME   "FYSETC AIO II"
-#define BOARD_WEBSITE_URL "fysetc.com"
+#define BOARD_WEBSITE_URL "github.com/FYSETC/FYSETC-AIO_II"
 
 #define BOARD_NO_NATIVE_USB
 #define RESET_STEPPERS_ON_MEDIA_INSERT
 #define DISABLE_JTAG
 
-#define pins_v2_20190128                          // new pins define
+#define PINS_V2_20190128                          // new pins define
```

Comment:
I was wary of you changing this in case people are overriding it...but it is not set up to be overridden.

I'm guessing this is just clutter from some early prototypes or something... If that is the case ideally we could just deleted it and all the code that it was hiding.

Reply:
It's been there since day one of the pins file, so I'm not sure why it exists.